dubbo-rpc/dubbo-rpc-rest/src/main/java/org/apache/dubbo/rpc/protocol/rest/RestProtocol.java

```diff
@@ -113,6 +122,29 @@ public void afterUnExport() {
         return exporter;
     }
 
+    private void checkJsonCompatibility(Class<?> clazz, String jsonCheckLevel) throws RpcException {
+
+        if (jsonCheckLevel == null || JSON_CHECK_LEVEL_WARN.equals(jsonCheckLevel)) {
+            boolean compatibility = JsonCompatibilityUtil.checkClassCompatibility(clazz);
+            if (!compatibility) {
+                List<String> unsupportedMethods = JsonCompatibilityUtil.getUnsupportedMethods(clazz);
+                assert unsupportedMethods != null;
+                logger.warn("", "", "", String.format("Interface %s does not support json serialization, the specific methods are %s.", clazz.getName(), unsupportedMethods));
+            } else {
+                logger.debug("Check json compatibility complete, all methods of {} can be serialized using json.", clazz.getName());
+            }
+        } else if (JSON_CHECK_LEVEL_STRICT.equals(jsonCheckLevel)) {
+            boolean compatibility = JsonCompatibilityUtil.checkClassCompatibility(clazz);
+            if (!compatibility) {
+                List<String> unsupportedMethods = JsonCompatibilityUtil.getUnsupportedMethods(clazz);
+                assert unsupportedMethods != null;
+                throw new RpcException(String.format("Interface %s does not support json serialization, the specific methods are %s.", clazz.getName(), unsupportedMethods));
```

Comment:
change `RpcException` to `IllegalStateException`.

Reply:
done